pom.xml

```diff
@@ -29,6 +29,11 @@
       <artifactId>argparse4j</artifactId>
       <version>0.8.1</version>
     </dependency>
+    <dependency>
+      <groupId>org.mockito</groupId>
+      <artifactId>mockito-core</artifactId>
+      <version>3.2.4</version>
+    </dependency>
```

Comment:
Can you change the dependency to the following?
```
        <dependency>
            <groupId>org.mockito</groupId>
            <artifactId>mockito-core</artifactId>
            <version>3.4.4</version>
            <scope>test</scope>
        </dependency>
```

Reply:
Done =)